homeassistant/components/bang_olufsen/strings.json

```diff
@@ -61,6 +77,10 @@
         "beolink_jid": {
           "name": "Beolink JID",
           "description": "Manually specify Beolink JID to join."
+        },
+        "source_id": {
+          "name": "Source",
+          "description": "Specify which source to join, behavior varies between hardware platforms. Source names prefaced by a platform name can only be used when connecting to that platform. For example \"ASE Beoradio\" can only be used when joining an ASE device, while ”ASE / Mozart Deezer” can be used with ASE or Mozart devices. A defined Beolink JID is required."
```

Comment:
Why does this sound so complicated? How is this displayed in the Bang & Olufsen app? Is this logical for the average user? Can we use validation to make sure this works well?

Reply:
The Beolink Converter NL/ML is a legacy product, designed to link the old B&O multiroom system from the 90’s to the current one. It is considered an advanced product, and the average user does not have one.
It does however enable current B&O products to link to legacy  sources, something that the B&O app also allows.
There is input validation through HA and if the source can't be joined, then the device will give an audible error indication.